app/assets/stylesheets/application.css.scss

```diff
@@ -203,4 +227,28 @@ a.btn.btn-default:hover {
   .small_screen_h3 {
     font-size: 10px;
   }
+
+  .picture_background {
+    margin: -30px 5px 5px 5px;
+  }
+
+  .photo-text {
+    width: 100%;
+    margin-left: -12px;
+  }
+
+  .scholar {
+    background-image: none;
+    background-color: grey;
+  }
+
+  .gi-bill {
+    background-image: none;
+    background-color: grey;
```

Comment:
Color `grey` should be written in hexadecimal form as `#808080`<br>Color literals like `grey` should only be used in variable declarations; they should be referred to via variable everywhere else.